aes.go

```diff
@@ -56,11 +56,17 @@ func (c *aesCipher) BlockSize() int {
 }
 
 func (c *aesCipher) Encrypt(dst, src []byte) {
-	c.encrypt(dst, src)
+	if err := c.encrypt(dst, src); err != nil {
+		// Upstream expects that the panic message starts with "crypto/aes: ".
```

Comment:
Hmm, I suppose "upstream" is not necessarily clear inside a technically independent module like this.

```suggestion
		// The Go crypto packages expect that the panic message starts with "crypto/aes: ".
```

Reply:
Good point.